shared/common-adapters/rounded-box.tsx

```diff
@@ -8,13 +8,14 @@ const Kb = {
 
 type Props = {
   children: React.ReactNode
-  side: 'bottom' | 'middle' | 'top'
+  side?: 'bottom' | 'middle' | 'top'
```

Comment:
why make this optional

Reply:
You might want a fully rounded box (as I'm using on the “Do you know your password?” input).

Reply:
maybe just add 'all'?